sources/osg/Texture.js

```diff
@@ -809,6 +809,11 @@ utils.createPrototypeStateAttribute(
                     // image update like video
                     if (this._image !== undefined && this._image.isDirty()) {
                         this.applyImage(gl);
+                        // if it's a video
+                        // and filter are set to mipmap
```

Comment:
The comment is not great, in a general case, it's more.
if the image is dirty we need to update the texture and mipmap if a mipmap filter is used

Reply:
Indeed. 
either I remove the comment or I replace "video" by "animated image" ?

Reply:
you can remove the comment I think